tests/scripts/thread-cert/node.py

```diff
@@ -48,6 +48,7 @@ def __init__(self, nodeid, is_mtd=False, simulator=None, version=None):
         self.verbose = int(float(os.getenv('VERBOSE', 0)))
         self.node_type = os.getenv('NODE_TYPE', 'sim')
         self.env_version = os.getenv('THREAD_VERSION', '1.1')
+        self._initialized = False
```

Comment:
We can add `_initialized = False` as a class member of `Node`.

Reply:
Is this based on the truth that `Simulator` is used as a singleton?

Reply:
No. `self. _initialized` will simply gets `Simulator._initialized` if `_initialized` is not defined on the instance.

Reply:
This is minor. I am fine with both approach.